keyboards/flutterby/rules.mk

```diff
@@ -0,0 +1,34 @@
+# MCU name
+MCU = atmega32u4
+F_CPU = 8000000
+ARCH = AVR8
+F_USB = $(F_CPU)
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Feather uses the caterina bootloader
+OPT_DEFS += -DBOOTLOADER_SIZE=4096 -DCATERINA_BOOTLOADER
+
```

Comment:
Most of this is unneeded. 

And the bootloader can be set by adding:

```make
# Bootloader selection
#   Teensy       halfkay
#   Pro Micro    caterina
#   Atmel DFU    atmel-dfu
#   LUFA DFU     lufa-dfu
#   QMK DFU      qmk-dfu
#   atmega32a    bootloadHID
BOOTLOADER = caterina
```

This will specify the bootloader type, and set the size.